_posts/2020-10-07-#19843.md

```diff
@@ -10,14 +10,51 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+* The Bitcoin network uses `addr` messages to communicate addresses aka
+  locations of nodes. To read more about the P2P message, you can see these
+  [docs](https://en.bitcoin.it/wiki/Protocol_documentation#addr).
```

Comment:
maybe:

"The Bitcoin P2P network uses `addr` messages to communicate addresses, aka the locations of nodes. To read more about the `addr` message, see these"...

Reply:
Updated. Thanks Jon!